plugins/inputs/netflow/netflow_decoder.go

```diff
@@ -693,6 +701,20 @@ func (d *netflowDecoder) decodeValueIPFIX(field netflow.DataField) []telegraf.Fi
 		elementID = field.Type & (0x4000 ^ 0xffff)
 	}
 
+	// Handle messages with Private Enterprise Numbers (PENs)
+	if field.PenProvided {
+		key := fmt.Sprintf("%d.%d", field.Pen, elementID)
+		// REMOVE
+		d.Log.Debugf("decoding PEN field %+v (eid: %d; key: %q)", field, elementID, key)
```

Comment:
Is this in here to aid debugging and were you going to remove it once you had confirmation from the user?

Reply:
This one is for removal as it is debugging only...